### PR TITLE
Add Prometheus to list of other ROS1 resources

### PR DESCRIPTION
**Pull request by potato77**

### CODE REVIEW

**Comment by hamishwillee** [comment]
@TSC21 This looks good - any comments/questions before I merge?

**Comment by hamishwillee** [comment]
@TSC21 ?

**Comment by TSC21** [comment]
I don't know the project so I can't really comment on it and neither I have a strong opinion. I just wonder if the idea is to add any similar project that gets created with similar features to this list. My opinion is that this shouldn't be the sort of things we add to the guide, specially because those references are off our control and we don't control their content and quality.

**Comment by TSC21** [comment]
@mrpollo tagging you to raise your awareness to the above.

**Comment by mrpollo** [comment]
Hey @TSC21 and @hamishwillee, this project is by DF Silver Member AMOVLab. They have posted about it numerous times on Slack but never really got much attention. I think they need our help to get it out there to the community, and I think adding it to the documentation is a good first step. I will work with them to promote it to the community.

To be clear, this is a mature project that has gained good adoption within the Chinese community, and they are hoping more developers from other locations take advantage of the open-source packages.

**Comment by TSC21** [comment]
> Hey @TSC21 and @hamishwillee, this project is by DF Silver Member AMOVLab. They have posted about it numerous times on Slack but never really got much attention. I think they need our help to get it out there to the community, and I think adding it to the documentation is a good first step. I will work with them to promote it to the community.
> 
> To be clear, this is a mature project that has gained good adoption within the Chinese community, and they are hoping more developers from other locations take advantage of the open-source packages.

Thanks for bringing clarity. Works for me!

**Review by TSC21 — APPROVED** [review]

File: en/ros/ros1.md [hunk 1]
```diff
@@ -23,3 +23,6 @@ The main topics covered are:
 - [PX4 ROS Setups](../ros/README.md#ros-setups).
 - [XTDrone](https://github.com/robin-shaun/XTDrone/blob/master/README.en.md) - ROS + PX4 simulation environment for computer vision.
   The [XTDrone Manual](https://www.yuque.com/xtdrone/manual_en) has everything you need to get started!
+- [Prometheus Open Source Autonomous Drone Project](https://github.com/amov-lab/Prometheus/blob/master/README_EN.md) - Prometheus is an open source autonomous drone software platform, which provides full set of solutions for the intelligent and autonomous flight of drones.
```
Comment (mrpollo) [inline en/ros/ros1.md:26]:
```suggestion
- [Prometheus Autonomous Drone Project](https://github.com/amov-lab/Prometheus/blob/master/README_EN.md) - Prometheus is a ROS 1 based, BSD-3 licensed collection of autonomous drone software packages from [AMOVLab](https://github.com/amov-lab), which provides a full set of solutions for the intelligent and autonomous flight of drones, such as mapping, localization, planning, control, and target detection, fully integrated with the Gazebo Simulator.
```

File: en/ros/ros1.md [hunk 2]
```diff
@@ -23,3 +23,6 @@ The main topics covered are:
...
+  This project is based on the PX4 Autopilot, and aims to provide PX4 developers with mature airborne computer terminal programs, bringing a more concise and faster development experience.
```
Comment (mrpollo) [inline en/ros/ros1.md:27]: This sentence is redundant, and I think we can remove it.

File: en/ros/ros1.md [hunk 3]
```diff
@@ -23,3 +23,6 @@ The main topics covered are:
...
+  At present, modules such as mapping, localization, planning, control and target detection have been integrated, along with Gazebo simulation.
```
Comment (mrpollo) [inline en/ros/ros1.md:28]: Let's remove and add the above.

**Review by mrpollo — CHANGES_REQUESTED** [review]
I think my version is more concise and brings more clarity into the documentation entry

**Comment by hamishwillee** [comment]
@mrpollo I have accepted your revisions (good) and merged.

@TSC21 Nuno, good questions. FWIW as long as it is clearly separated from the "PX4/core ROS offering" and as long as it is an accurate description (i.e. not "over-marketed") then I have no problem with supporting/enabling technologies like this being linked from our docs. I share your concerns that this might not be all that effective, but as Ramon is working with them to help them out all good with me.

**Pull request merged**